src/figuresClasses.ts

```diff
@@ -1,11 +1,94 @@
-export interface Figure {}
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
```

Comment:
[CODE STYLE] - Enum constant names usually start with a capital letter. Please update the enum members to start with a capital letter to follow best practices. For example: `Triangle = 'triangle'` should be `Triangle = 'TRIANGLE'`.